assets/terraform-modules/aws/flatcar-linux/kubernetes/ami.tf

```diff
@@ -1,37 +1,11 @@
 locals {
-  # Pick a Container Linux derivative
-  ami_id = element(concat(data.aws_ami.flatcar.*.image_id, data.aws_ami.coreos.*.image_id, list("")), 0)
+  ami_id = data.aws_ami.flatcar.*.image_id
```

Comment:
We can skip `local.ami_id` completely and use `data.aws_ami.flatcar.image_id` directly.

Reply:
Ah yes, whoops. Thanks. Fixed.